redash/serializers/__init__.py

```diff
@@ -294,14 +294,15 @@ def serialize_job(job):
         status = 4
     else:
         error = ""
-        query_result_id = job.result
+        result = query_result_id = job.result
 
     return {
         "job": {
             "id": job.id,
             "updated_at": updated_at,
             "status": status,
             "error": error,
+            "result": result,
             "query_result_id": query_result_id,
```

Comment:
To avoid accumulating tech debt, maybe we can return `query_result_id` only when it's a `execute_query` task?

Reply:
(only if it's simple check)

Reply:
If we want to avoid accumulating tech debt, we should probably always return `result` and ditch `query_result_id`, no?

Reply:
But this will break scripts that use /jobs for polling and expect `query_result_id`... :| We could really use API versioning at this point.

Reply:
I agree, that's why I wen't for doubling `result` and `query_result_id`, as the cheapest thing that won't break and won't be too awkward :(